### DISCUSSION TITLE
Can't upload to import_files

### DISCUSSION THREAD

**Discussion by MarkLeece** [iterate-ch/cyberduck#16088] [Q&A]
I'm trying to upload a csv file (backorder2906.csv) to the import_files directory of my bigcommerce store. At first it wouldn't successfully upload and would say there was an error at the end and then continuously retry. I uninstalled and reinstalled the program and now it says its successfully uploaded the file but the file doesn't exist in the folder. I've tried twice but it's not working.

![image](https://github.com/iterate-ch/cyberduck/assets/174163833/461d1c89-63b6-4dcc-bb4e-026cc124bc42)

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-16088 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
